modules/apps/commerce/commerce-subscription-web/src/main/java/com/liferay/commerce/subscription/web/internal/display/context/CommerceSubscriptionEntryDisplayContext.java

```diff
@@ -284,6 +284,10 @@ public String getOrderPaymentMethodImage() throws PortalException {
 				fetchCommercePaymentMethodGroupRel(
 					commerceOrder.getGroupId(), paymentMethodKey);
 
+		if (commercePaymentMethodGroupRel == null) {
+			return StringPool.BLANK;
+		}
```

Comment:
Also this should not happen if the subscription is possible only when a payment method (with subscription enabled) is selected